### DISCUSSION TITLE
Sample configure script doesn't work

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#28] [General]
I took the sample configure.in from http://bsdbuild.hypertriton.com/faq.html and properly created the configure script. But when I executed this script in MSYS2, it said:

```
$ ./configure
BSDBuild 3.2 <https://bsdbuild.hypertriton.com/>
* Overwriting existing Makefile.config
checking for a C compiler...yes, /usr/bin/cc
checking whether the C compiler works...yes, it outputs .exe files
cc: checking whether compiler is Clang...no
cc: checking whether compiler is cc65...no
cc: checking whether compiler is emcc...no
cc: checking for compiler warning options...yes
cc: checking whether compiler is GCC...yes
cc: checking for float and double...yes
cc: checking for long double...yes
cc: checking for long long...yes
./configure: line 1359: syntax error near unexpected token `else'
./configure: line 1359: `else'
```

Here is the generated configure script:
[configure.txt](https://github.com/JulNadeauCA/BSDBuild/files/14625246/configure.txt)

BSDBuild is the latest source from Github.

**Comment by JulNadeauCA** [comment]
The error on line 1359 is caused by the statement:
`if [ "${HAVE_CC_ASM}" = "yes" ]; then`
`else`
`fi`

This is due to there being no `c_incdir_config()` or `c_include_config()` statement in your `configure.in`.
If you call either, there will be an output defined and you should not longer get this error.

Good find! It's kind of an oddball case, but I'll add a check for it shortly.

**Comment by JulNadeauCA** [comment]
I've updated [the FAQ](https://bsdbuild.hypertriton.com/faq.html) example to include the missing directive.

**Comment by JulNadeauCA** [comment]
Fixed in https://github.com/JulNadeauCA/BSDBuild/commit/9bb96d74abce957a26a448860c7de7936df80640.
